.doc_gen/snippet_files/aurora_Get-RDSOrderableDBInstanceOption.1.1.ps1

```diff
@@ -0,0 +1,5 @@
+Get-RDSOrderableDBInstanceOption `
+  -Engine aurora-postgresql `
+  -DBInstanceClass db.r5.large `
+  -Region us-east-1  | `
+Format-Table EngineVersion, SupportedEngineModes
```

Comment:
Using `Format-Table` is not required so it might be better to leave it out. Again, granted, we need to clean up some other existing examples that are using format cmdlets.